docs/real-time/azure-signalr-scenario.md

```diff
@@ -268,10 +270,33 @@ builder.Services.AddSingleton(sp =>
 
 var app = builder.Build();
 
+app.MapPost("/negotiate", async (string? userId, ServiceManager sm, CancellationToken token) =>
+{
+    // The creation of the ServiceHubContext is expensive, so it's recommended to 
+    // only create it once per named context / per app run if possible.
+    var context = await sm.CreateHubContextAsync("messages", token);
+    
+    NegotiationResponse negotiateResponse = await n.NegotiateAsync(new NegotiationOptions
+    {
+        UserId = userId,
+        ClientType = ClientType.JavaScript,
+        HubName = "messages"
+    }, token);
+    
+    return Results.Json(negotiateResponse, new JsonSerializerOptions(JsonSerializerDefaults.Web)
```

Comment:
`Results.Ok(negotiateResponse)` is enough?

Reply:
No, there's a bug that prevents that from working in .NET: see https://github.com/dotnet/aspnetcore/issues/60935